apps/partners/src/state/appState.ts

```
@@ -0,0 +1,17 @@
+import { create, StateCreator } from 'zustand'
+
+interface Organization {
+  id: string
+  name: string
+  [key: string]: any
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)